packages/react-instantsearch-hooks-web/src/widgets/RefinementList.tsx

```diff
@@ -94,6 +107,16 @@ export function RefinementList({
     }
   }
 
+  const mergedTranslations = {
+    resetButtonTitle: 'Clear the search query.',
+    submitButtonTitle: 'Submit the search query.',
```

Comment:
We didn't add periods in the RFC, and I don't think it fits well with button labels. wdyt?

```suggestion
    resetButtonTitle: 'Clear the search query',
    submitButtonTitle: 'Submit the search query',
```